ext/message_admin/managed/SavedSearch_System_Workflow_Messages.mgd.php

```diff
@@ -1,5 +1,11 @@
 <?php
 
+require_once 'message_admin.civix.php';
+// phpcs:disable
+use CRM_MessageAdmin_ExtensionUtil as E;
+// phpcs:enable
+
```

Comment:
I don't think the require once or the phpcs comments are needed. I don't see them in any other `.mgd.php` files

Reply:
Gone now